src/little_dict.jl

```diff
@@ -54,7 +54,7 @@ LittleDict(ks, vs) = LittleDict(collect(ks), collect(vs))
 function LittleDict{K,V}(itr) where {K,V}
     ks = K[]
     vs = V[]
-    for val in itr
+   for val in itr
```

Comment:
😂  idk how one space got deleted here

```suggestion
    for val in itr
```